internal/terminal/list.go

```diff
@@ -26,6 +26,14 @@ func newList(message string, data []interface{}) list {
 }
 
 func (l list) Message() (string, error) {
+	if len(l.data) == 0 {
+		return l.message, nil
+	}
+
+	if len(l.data) == 1 {
+		return fmt.Sprintf("%s: %s", l.message, l.data[0]), nil
```

Comment:
let's add some logic within this case here to account for a potentially empty `l.message` (the other cases don't need that logic just due to the way its displayed)